Cataclysm/Druid.lua

```diff
@@ -2369,10 +2427,9 @@ spec:RegisterAbilities( {
         gcd = "totem",
 
         spend = function()
-            local unglyphed = (buff.clearcasting.up and 0) or (25 * ((buff.berserk.up and 0.5) or 1))
-            --Glyph of Ferocious Bite: and consumes up to 25 additional energy to increase damage by up to 100%, and heals you for 1% of your total maximum health for each 10 energy used
-            local additional_due_glyph = (glyph.ferocious_bite.enabled and min(25, energy.current - unglyphed) or 0)
-            return  unglyphed + additional_due_glyph end, 
+            local base_cost = (buff.clearcasting.up and 0) or (25 * ((buff.berserk.up and 0.5) or 1))
+            local excess_energy = min(25, energy.current - base_cost) or 0
```

Comment:
Ah, I read the glyph wrong. So the excess are consumed all the time, only saw the part on the glyph:

> Glyph of Ferocious Bite
> and consumes up to 25 additional energy to increase damage by up to 100%, and heals you for 1% of your total maximum health for each 10 energy used
> 
> and consumes up to 25 additional energy to increase damage by up to 100%.  Damage is increased by your attack power.